sale_timesheet_invoice_link/readme/USAGE.md

```diff
@@ -0,0 +1,6 @@
+To use this module, you need to:
+
+1. Go to *Timesheets / Timesheets / All Timesheets*.
+1. Select the list view.
```

Comment:
```suggestion
2. Select the list view.
```

Reply:
I personally prefer to just use `1.` always, because this way you don't have to care if in the future another line lands in between. Markdown renders it just fine. It's the equivalent to restructuredtext's `#.`